### ISSUE TITLE
Output minimum structural entropy

### ISSUE THREAD

**Issue by LHXqwq** [zengguangjie/deDoc2#2]
Can I also output the value of the minimum structural entropy after the iteration to the file?

**Comment by zengguangjie** [comment]
yes, you can use the function public double calStrucEntro2D(ArrayList<ArrayList<Integer>> partition) in deDoc2/deDoc2/src/algorithm/DPAlgorithm.java. This function returns the 2-D structural entropy given the calculated graph partitioning (i.e., the segmented TADs).
